packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -38,6 +38,7 @@ describe( 'getSupportedPlans', () => {
 				product_slug: PLAN_PREMIUM_MONTHLY,
 				raw_price: 13,
 				product_id: 3,
+				bill_period: 31,
```

Comment:
We should remove the mention of the decimal places from the comment (a few lines above), now that this comes from an external package.

Maybe the comment should be changed to "Premium plan, monthly billed"